tests/phpunit/tests/html-api/wpHtmlTagProcessor-token-scanning.php

```diff
@@ -760,6 +760,126 @@ public function test_basic_assertion_funky_comment() {
 		);
 	}
 
+	/**
+	 * Ensures that various funky comments are properly parsed.
+	 *
+	 * @ticket 60170
+	 *
+	 * @since 6.6.6
```

Comment:
Wrong version looks like:
```suggestion
	 * @since 6.6.0
```

Reply:
Fixed in d374d5774d68357220fa46f5f991ab8338f163d3